src/config.cpp

```diff
@@ -62,6 +62,19 @@ struct ProviderOptionsArray_Element : JSON::Element {
   ProviderOptionsObject_Element object_{v_};
 };
 
+GraphOptimizationLevel getGraphOptimizationLevel(std::string_view name) {
```

Comment:
To keep in style with the casing used in other methods within this file:

```suggestion
GraphOptimizationLevel GetGraphOptimizationLevel(std::string_view name) {
```